src/app/site/mobile-header/mobile-header.component.ts

```diff
@@ -14,11 +14,10 @@ export class MobileHeaderComponent implements OnInit {
   isMobile = false;
   sideNavIsOpen = false;
   array: any = [];
-  @Input() menuWidth = 100;
-  @Input() menuHeight = 100;
 
```

Comment:
This is fine. Because `height: 100vh` is equal to `[styles.height]="curHeight"`.
It was redundancy to make an input from another component to this component just for setting the height and width, as we can simply use `100vh` to fill as high as the view height. just check the `fixed` class in related `css` file to see what I mean.